### PR TITLE
Delta compute

### PR DESCRIPTION
**Pull request by tok-kkk**

### CODE REVIEW

**Comment by loongy** [comment]
Majority of missing test coverage is not being utilized right now so not too worried about this.

File: node_test.go [hunk 1]
```diff
@@ -1,12 +1,134 @@
...
-	// . "github.com/onsi/gomega"
+	. "github.com/onsi/gomega"
+	"github.com/republicprotocol/go-dark-node"
+	"github.com/republicprotocol/go-order-compute"
+	"github.com/republicprotocol/go-rpc"
+)
+
+const (
+	Number_Of_Bootstrap_Nodes = 4
+	Number_Of_Test_NODES      = 4
```
Comment (loongy) [inline node_test.go]: Is there are reason this is not using the standard CamelCase?

File: node_test.go [hunk 2]
```diff
@@ -1,12 +1,134 @@
...
+		})
+
+		It("should be able to run startup successfully", func() {
+			for _, n := range nodes {
+				go func(n *node.DarkNode) {
+					Ω(n.Start()).ShouldNot(HaveOccurred())
+				}(n)
+			}
+			chanInt := make(chan int)
+			<- chanInt
```
Comment (loongy) [inline node_test.go]: Does this block forever? Reading from a synchronous channel with no writer.

File: node_test.go [hunk 3]
```diff
@@ -1,12 +1,134 @@
...
+				Ω(err).ShouldNot(HaveOccurred())
+				for i := range shares {
+					if err := rpc.SendOrderFragmentToTarget(nodes[i].Configuration.MultiAddress, nodes[i].Configuration.MultiAddress.Address(), nodes[0].Configuration.MultiAddress, shares[i], 5*time.Second); err != nil {
+						log.Fatal(err)
+					}
+				}
+			}
+			time.Sleep(5 * time.Second)
+
+			time.Sleep(time.Minute)
```
Comment (loongy) [inline node_test.go]: Combine these sleeps. No reason to have them separated.

File: node_test.go (hunk 2 above)
Comment (tok-kkk) [reply node_test.go]: They were there for testing the `register` function which usually takes several minutes , forget to remove it.  Have fixed it

File: node_test.go (hunk 1 above)
Comment (tok-kkk) [reply node_test.go]: Fixed

File: node_test.go (hunk 3 above)
Comment (tok-kkk) [reply node_test.go]: Fixed

**Pull request merged**